Shopping.java

```diff
@@ -0,0 +1,190 @@
+package Lession3SeleniumWebPackage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.Select;
+
+/**
+ * Chọn 1 số sản phẩm và đặt mua
+ * 
+ * @author ThaoBTP
+ *
+ */
+public class Shopping {
+	public static WebDriver driver = null;
+
+	public static String homePage = "http://live.guru99.com/";
+	public static String accountPage = "http://live.guru99.com/index.php/checkout/onepage/success/";
+	public static String soLuong = "4";
+
+	public static String tvMenuLocator = "xpath=.//*[contains(text(),'TV')]";
+	public static String productLocator = "xpath=.//*[@id='product-collection-image-4']";
+	public static String quantityLocator = "id=qty";
+	public static String addToCartLocator = "xpath=.//*[@title='Add to Cart']";
+	public static String country = "Vietnam";
+	public static String province = "HaNoi";
+	public static String blFristName = "first name";
+	public static String blLastName = "last name";
+	public static String blCompany = "company";
+	public static String blEmail = "email@gmail.com";
+	public static String blAddress = "address";
+	public static String blStAddress = "street address 2";
+	public static String blCity = "city";
+	public static String blProvince = "Alaska";
+	public static String blZip = "zip";
+	public static String blCountry = "Vietnam";
+	public static String blTel = "telephone";
+	public static String blFax = "fax";
+
+	public static String countryLocator = ".//*[@id='country']";
```

Comment:
Should add comments for each part of element. for example: billing informaion, shopitem, ...